acceptance/tests/resource/file/should_create_symlink.rb

```diff
@@ -2,10 +2,10 @@
 
 message = 'hello world'
 agents.each do |agent|
-  confine_block :to, :platform => 'windows' do
+  if agent.platform.variant == 'windows'
```

Comment:
Only thing I can find is the commit it was introduced in last July (https://github.com/puppetlabs/beaker/commit/10633f79f5fd6efd3c3fe5d6ee234fc54c5df4b1),

```
# A string with the name of the platform.
```